<launch>

    <param name="initial_pose_x" value="0.0" />
    <param name="initial_pose_y" value="0.0" />
    <param name="initial_pose_theta" value="0.0" />
    <param name="N" value="40" />
    <param name="R" value="0.06844" />
    <param name="LW" value="0.36279" />

    <node pkg="project1" type="odom_pub" name="odom_pub" />
    <node pkg="project1" type="vel_pub" name="vel_pub" />
    <node pkg="project1" type="inverter" name="inverter" />
    <node pkg="project1" type="pose_broadcaster" name="pose_broadcaster" />

</launch>
